bolt-design-system/bolt
# Attributes fix for all list components

## Jira

http://vjira2:8080/browse/BDS-1094

## Summary

Fix attributes issue that exists only when the list has empty elements.

## How to test

Please create a new component using `npm run cc` then remove readme.md from the component directory and check if on docs page of this component we don't have js errors and in right top component menu is without extra empty elements.
![image](https://user-images.githubusercontent.com/16049049/58555688-63678500-821a-11e9-92f1-65b30798ecc3.png)

Check if all test is passing of list elements.
